app/move/views/view.njk

```diff
@@ -22,6 +22,7 @@
   }) }}
 
   <a href="javascript:window.print()" class="app-!-float-right app-print--hide govuk-!-margin-top-3">
+    {{ appIcon({ id: "print-icon" }) }}
```

Comment:
Might be better to try and stick to BEM naming convention and just passing in `print` or `download` here.

Then in the classnames on the element it would build `app-icon app-icon--print`. At the moment it displays as `app-icon print-icon` which feels less consistent with the rest of the classname structure.